source/index.ts

```diff
@@ -16,6 +16,7 @@ export {got};
 export {default as Options} from './core/options.js';
 export * from './core/options.js';
 export * from './core/response.js';
+export {default as Request} from './core/index.js';
```

Comment:
```suggestion
export type {default as Request} from './core/index.js';
```

?

I don't think we want to export the actual class as that's an implementation detail.

Reply:
Ah yes, fine with me, thanks 🙇

Reply:
@sindresorhus is right - currently `Request` is only meant to be used as an interface. Got does a bit more than just `new Request`:

https://github.com/sindresorhus/got/blob/a405e7cb545ef56e8f3158ea9f14fffe8fb2da6e/source/create.ts#L57-L59